src/main/java/uk/gov/homeoffice/digital/sas/balancecalculator/client/AccrualsService.java

```diff
@@ -42,6 +44,9 @@ public AccrualsService(RestTemplateBuilder builder,
         accrualsUrl + "/resources/accruals?tenantId={tenantId}&filter={filter}";
     this.agreementsByIdUrl =
         accrualsUrl + "/resources/agreements/{agreementId}?tenantId={tenantId}";
+    this.impactedAccrualsUrl =
+        accrualsUrl + "/resources/accruals?tenantId={tenantId}&timeEntryId={timeEntryId}"
+          + "&timeEntryStartDate={timeEntryStartDate}&agreementEndDate={agreementEndDate}";
```

Comment:
This needs updating to match the updated function. We don't use agreementEndDate anymore. We use timeEntryEndDate